### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for edit in "Azure Signin dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1382]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to "Authentication file" edit and observe the name property using inspect.

**Actual Result:**
Name property is empty for the edit box

**Narrator behaviour:**
Edit ,read-only
blank

**NVDA behaviour:**
read-only

**Expected Result:**
Name property for the edit should be "Authentication file".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![azure signin-edit-no name](https://user-images.githubusercontent.com/22956130/35681627-f7409932-0783-11e8-8017-2810bf10fa6a.jpg)
![azure signin-edit-no name-narrator](https://user-images.githubusercontent.com/22956130/35681632-fb060002-0783-11e8-9912-03051564cb87.jpg)

**Comment by Eskibear** [comment]
already shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1382_fixed](https://user-images.githubusercontent.com/22956130/39762381-20d92736-52f8-11e8-8769-62542257cc0b.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749855
